linode/manager
# upcoming: [M3-8267] - Add Marketplace Cluster pricing support to Linode Create v2

## Description 📝

Adds support for Marketplace App Cluster pricing on Linode Create v2 💵 

## Preview 📷

![Screenshot 2024-06-28 at 1 10 15 PM](https://github.com/linode/manager/assets/115251059/fe329c86-a48a-47d8-843b-412fda1c7bfd)

## How to test 🧪

### Prerequisites
- Check out the PR and turn on Linode Create v2  (or use preview link with Linode Create v2 turned on)

### Verification steps
- Select a marketplace app that has a `Cluster` clip 🖱️ 
- Select a cluster size (if there is no default cluster size) 🔢 
- Select a region 🌎 
- Select a plan 📝 
- Go to the summary at the Botton of the page ⬇️ 
- Verify the summary shows the correct number of nodes and price 💰 
  - Perform the same steps on Linode Create v1 in another tab and compare  👀 

## As an Author I have considered 🤔
